core/modules/dashboard/includes/block.welcome.inc

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * @file
+ * Dashboard block providing a welcome message, and links to get people started
+ * using Backdrop.
+ */
+class DashboardWelcomeBlock extends Block {
+  /**
+   * {@inheritdoc}
+   */
+  function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Welcome to Backdrop!');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function getAdminTitle() {
+    return t('Welcome to Backdrop!');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function getAdminPreview() {
+    return t('Welcome message, and links to getting-started tasks.');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function getContent() {
+    $build = array(
+      '#theme' => 'dashboard_panel__welcome',
+    );
+    $build['welcome'] = array(
+      '#type' => 'help',
+      '#markup' => t('Here are some links to get you started:'),
+    );
+
+    $build['steps'] = array(
+      '#type' => 'container',
+      '#attributes' => array(
+        'class' => array('dashboard-welcome-steps', 'row'),
+      ),
+    );
+
+    $get_started_links = array();
+    $get_started_links['<front>'] = t('View your site');
+    $get_started_links['admin/appearance/settings/basis'] = t('Customize the look of your site');
+    $get_started_links['admin/structure/layouts/manage/home'] = t('Configure the layout of your home page');
+    $get_started_links['admin/appearance'] = t('Change the theme used by your site');
+    $get_started_links['admin/appearance/install'] = t('Find more themes to install');
+    $get_started_links = $this->filterAvailableLinks($get_started_links);
+
+    if ($get_started_links) {
+      $welcome_start_content = theme('item_list', array('items' => $get_started_links));
+      $build['steps']['start'] = array(
+        '#type' => 'container',
+        '#attributes' => array(
+          'class' => array('dashboard-welcome-step', 'dashboard-welcome-start', 'col-md-4'),
+        ),
+      );
+      $build['steps']['start']['content'] = array(
+        '#type' => 'markup',
+        '#prefix' => '<h4>' . t('Get started') . '</h4>',
+        '#markup' => $welcome_start_content,
+      );
+    }
+
+    $next_steps_links = array();
+    $next_steps_links['node/add/post'] = t('Write your first article');
```

Comment:
We already have the "Your first post!" article - and we should not call it "article", as @jenlampton also pointed during the previous dev meeting:

```suggestion
    $next_steps_links['node/add/post'] = t('Create a new post');
```